source/predicates/data-view.ts

```diff
@@ -0,0 +1,48 @@
+import {Predicate, PredicateOptions} from './predicate';
+
+export class DataViewPredicate extends Predicate<DataView> {
+	/**
+	@hidden
+	*/
+	constructor(options?: PredicateOptions) {
+		super('DataView', options);
+	}
+
+	/**
+	Test a DataView to have a specific byte length.
+
+	@param byteLength - The byte length of the DataView.
```

Comment:
It's inconsistent that the others use `array buffer` and `typed array`, while here it's `DataView`.

Reply:
I was just following your convention you used [here](https://github.com/sindresorhus/ow/blob/master/source/predicates.ts#L119). Since `ArrayBufferPredicate` was used by `ArrayBuffer` and `SharedArrayBuffer`, I use “array buffer” to represent them.